id.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version:       2.4
 
 name:                id
-version:             0.1.0.0
+version:             0.2.0.0
```

Comment:
I don't think we should bump version numbers unless we have this released on hackage.

When and if we end up releasing this to package it would be confusing to have some version only be present in git.